### ISSUE TITLE
Recurring sales invoice fails on v1.5.7

### ISSUE THREAD

**Issue by billott323** [ledgersmb/LedgerSMB#2933]
Clicking on Process Transactions under recurring transaction involving a sale invoice yields this error
```
Posting Sales Invoice 904 ..... done
Sending Invoice 904Status: 500 ISE Content-Type: text/html; charset=utf-8
Error!

'Invalid format at /opt/ledgersmb/tools/../lib/LedgerSMB/Template.pm line 227. '

dbversion: 1.5.7, company: Salem Street North, LLC
```
The invoice is NOT posted, even though the error message says its done. The only recovery is to go to the login page and login.

**Comment by ehuelsmann** [comment]
Closed by merging #2951 and #2952. Please confirm.

**Comment by billott323** [comment]
I believe it is fixed. Thank You!
